CryptoWallet/Scenes/Home/Wallet/WalletViewController.swift

```diff
@@ -7,9 +7,136 @@
 //
 
 import UIKit
+import SwiftChart
 
 final class WalletViewController: UIViewController {
+    @IBOutlet private weak var walletNameLabel: UILabel!
+    @IBOutlet private weak var totalValueLabel: UILabel!
+    @IBOutlet private weak var value24hChangeLabel: UILabel!
+    @IBOutlet private weak var assetSearchBar: UISearchBar!
+    @IBOutlet private weak var assetListTableView: UITableView!
+    @IBOutlet private weak var assetValueView: UIView!
+    @IBOutlet private weak var sixHoursChartLabel: UIButton!
+    @IBOutlet private weak var oneDayChartLabel: UIButton!
+    @IBOutlet private weak var oneWeekChartLabel: UIButton!
+    @IBOutlet private weak var oneMonthChartLabel: UIButton!
+    @IBOutlet private weak var oneYearChartLabel: UIButton!
+    @IBOutlet private weak var allChartLabel: UIButton!
+    @IBOutlet weak var assetPriceChartLabel: UILabel!
+    
+    private var assetList = [AssetInfo]()
+    
     override func viewDidLoad() {
         super.viewDidLoad()
+        configView()
+    }
+    
+    private func configView() {
+        [oneDayChartLabel,
+         oneWeekChartLabel,
+         oneMonthChartLabel,
+         oneYearChartLabel,
+         allChartLabel].forEach {
+            $0?.setTitleColor(.darkGray, for: .normal)
+        }
+        let chart = Chart(frame: assetValueView.bounds)
+        chart.delegate = self
+        let data = [
+            (x: 1.1, y: 160.51),
+            (x: 1.5, y: 158.3),
+            (x: 2.3, y: 155.32),
+            (x: 3.1, y: 156.27),
+            (x: 4, y: 158.44),
+            (x: 5.5, y: 161.31),
+            (x: 6.1, y: 163.23),
+            (x: 7.4, y: 166.85),
+            (x: 8.2, y: 169.25),
+            (x: 9.1, y: 175.42),
+            (x: 9.7, y: 175.02),
+            (x: 10.8, y: 174.32)
+        ]
+        let series = ChartSeries(data: data)
+        series.area = true
+        let xLabels = ChartHelper.findLabels(data: data, axis: .X, numberLabel: 6)
+        chart.xLabels = xLabels
+        chart.xLabelsFormatter = { String(Int(round($1))) + "h" }
+        let yLabels = ChartHelper.findLabels(data: data, axis: .Y, numberLabel: 6)
+        chart.yLabels = yLabels
+        chart.yLabelsFormatter = { "$" + String(Int($1)) }
+        chart.add(series)
+        chart.hideHighlightLineOnTouchEnd = true
+        assetValueView.addSubview(chart)
+        assetListTableView.do {
+            $0.delegate = self
+            $0.dataSource = self
+            $0.rowHeight = 60
+            $0.estimatedRowHeight = UITableView.automaticDimension
+            $0.tableFooterView = UIView()
+        }
+        let ethereum = AssetInfo(assetLogo: UIImage(named: "ethereum"),
```

Comment:
tất cả dữ liệu test nên tạo mock r gọi, không tạo trực tiếp trên viewcontroller như này
```suggestion
       extension AssetInfo {
    static func mock() -> AssetInfo {
        return AssetInfo(assetLogo: UIImage(named: "ethereum"),
                    assetName: "Ethereum",
                    ....
                    )
    }
}
```

Reply:
lúc call dữ liệu fake thì gọi `AssetInfo.mock()`